### PR TITLE
BEAM-2122] Allow table descriptions to be null

### PR DESCRIPTION
**Pull request by reuvenlax**
Wrap the coder with a NullableCoder.

R: @jkff

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableDestinationCoder.java [hunk 1]
```diff
@@ -40,15 +45,15 @@ public void encode(TableDestination value, OutputStream outStream, Context conte
     if (value == null) {
       throw new CoderException("cannot encode a null value");
     }
-    stringCoder.encode(value.getTableSpec(), outStream, context.nested());
-    stringCoder.encode(value.getTableDescription(), outStream, context.nested());
+    tableSpecCoder.encode(value.getTableSpec(), outStream, context.nested());
+    tableDescriptionCoder.encode(value.getTableDescription(), outStream, context);
```
Comment (jkff) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableDestinationCoder.java]: .nested() was dropped here, but not below. Looks like a bug?

**Comment by jkff** [comment]
Will fix and merge this myself.

**Comment by reuvenlax** [comment]
retest this please

**Comment by davorbonaci** [comment]
Cherrypick?

**Pull request closed without merging**